modules/special-rules/pages/tyranid-objectives.adoc

```diff
@@ -0,0 +1,16 @@
+= Tyranid Objectives
+
+Always ready for a krumpin, Orks will often charge into melee before the enemy has even realised that the Orks are close enough to charge them!
+
+---
+
+Apply these rules to Tyranid armies:
+
+* The army does not use objectives, regardless of scenario.
+* If an enemy detachment is wiped out, add its morale value to the Tyranid player's army morale in the Rally phase. [either/or]
```

Comment:
```suggestion
* After you destroy an enemy detachment, add its morale value to your army morale in the Rally phase.
```

Reply:
As above, 'destroy' is effectively a reserved term for specific types of war engine deaths.

Reply:
I've added this to the commit batch, but I'm still not a fan of add/remove for army morale.

Reply:
This didn't get committed, somehow...